tensorflow/core/kernels/mkl_conv_ops.h

```diff
@@ -35,9 +35,9 @@ limitations under the License.
 #include "tensorflow/core/lib/strings/str_util.h"
 #include "tensorflow/core/platform/logging.h"
 #include "tensorflow/core/platform/macros.h"
+#include "tensorflow/core/util/mkl_util.h"
 #include "tensorflow/core/util/padding.h"
 #include "tensorflow/core/util/tensor_format.h"
-#include "tensorflow/core/util/mkl_util.h"
 #include "mkldnn.hpp"
```

Comment:
Please also move this to before `#include "tensorflow/core/framework/numeric_op.h"`.

Reply:
Not a problem. Actually it was my bad. I just did code change by adjusting header file inclusion order in all 3 changed files of this PR. Thanks!